root/entity/edits.tt

```diff
@@ -2,8 +2,8 @@
     SET type_for_css = ent_type;
     SET type_for_css = 'rg' IF ent_type == 'release_group';
 
-    SET name_for_title = html_escape(entity.name);
-    SET name_for_title = html_escape(entity.l_name) IF ent_type == 'instrument';
+    SET name_for_title = entity.name;
+    SET name_for_title = entity.l_name IF ent_type == 'instrument';
     SET name_for_title = display_url(entity) IF ent_type == 'url';
```

Comment:
This is actually slightly broken, too, but I’m replacing `display_url` in another branch anyway.